ONSdigital/design-system
# Fix checkboxes radios textboxes being auto focused on error summary link

### What is the context of this PR?
Fixes: #1047 

### How to review 
- Check that clicking on an error summary link no longer changes what you are selecting
- Check this doesn't affect any other checkboxes or radios
